### DISCUSSION TITLE
API to list projects is not working : Tried recreating a new token will all permissions..

### DISCUSSION THREAD

**Discussion by wquraishikpmg** [community/community#143312] [Apps, API and Webhooks]
### Body

Hello everyone,

I am experiencing an issue with the GitHub API for fetching organization projects. Here are the details:

**API Endpoint**:
`https://api.github.com/orgs/[Org_name]/projects`

**Request Details**:
- Method: GET
- Headers:
  - Authorization: token PERSONAL_ACCESS_TOKEN
  - Accept: application/vnd.github.inertia-preview+json

**Problem**:
Despite having active projects in the organization, the API response consistently **returns an empty array `[ ]**`.

**Attempts**:
- Tried using different personal access tokens with full `repo`, `read:org`, and `project` scopes.
- Used both the Postman web and desktop application.
- Confirmed that projects do exist in the organization by manually checking the GitHub UI.
- Even recreated a new token will all possible permissions enables in the new token 
**Questions**:
- Is there an additional permission or step missing to access organization projects?  
  (reference link API doc: https://docs.github.com/en/rest/projects/projects?apiVersion=2022-11-28#get-a-project)
- Has anyone faced a similar issue, and what was the resolution?
- Are there better ways to debug this issue further?

Any help or pointers would be greatly appreciated!

Thank you,
Wajahath

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @wquraishikpmg, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `API & Webhooks` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!
